CODEOWNERS

```diff
@@ -1,4 +1,5 @@
-# Until we have deccent coverage, and are roughly feature complete, 
-# I (Sunil) would like to approve every PR before it lands. 
+# Until we have decent coverage, and are roughly feature complete,
 
-*       @threepointone
+# I (Sunil) would like to approve every PR before it lands.
```

Comment:
Add yourself too, and remove this comment

Reply:
Done